libindy_vdr/Cargo.toml

```diff
@@ -17,13 +17,15 @@ crate-type = ["staticlib", "rlib", "cdylib"]
 
 [features]
 ffi = ["ffi-support", "logger"]
+did_indy = []
 logger = ["env_logger", "log"]
 zmq_vendored = ["zmq/vendored"]
 local_nodes_pool = []
 rich_schema = ["indy-data-types/rich_schema"]
 default = ["ffi", "log", "zmq_vendored"]
 
 [dependencies]
+chrono = "0.4.12"
```

Comment:
`chrono` is not supported anymore and we switched to `time`, see [commit here](https://github.com/hyperledger/indy-vdr/commit/d985f422b2d178b9bfcaa872bb0dff2f64a5c693)